### ISSUE TITLE
During refactoring the generated ee.jakarta.tck.persistence.core.EntityGraph.ClientPuservletTest code does not add `EntityGraph.ClientPuservletTest.class` in generated ShrinkWrap code

### ISSUE THREAD

**Issue by scottmarlow** [eclipse-ee4j/jakartaee-tck-tools#134]
Locally, I hacked the following change:

```
diff --git a/jpa/platform-tests/src/main/java/ee/jakarta/tck/persistence/core/EntityGraph/ClientPuservletTest.java b/jpa/platform-tests/src/main/java/ee/jakarta/tck/persistence/core/EntityGraph/ClientPuservletTest.java
index b9275c249..ee89d6234 100644
--- a/jpa/platform-tests/src/main/java/ee/jakarta/tck/persistence/core/EntityGraph/ClientPuservletTest.java
+++ b/jpa/platform-tests/src/main/java/ee/jakarta/tck/persistence/core/EntityGraph/ClientPuservletTest.java
@@ -75,6 +75,7 @@ public class ClientPuservletTest extends ee.jakarta.tck.persistence.core.EntityG
             ee.jakarta.tck.persistence.common.PMClientBase.class,
             com.sun.ts.tests.common.vehicle.servlet.ServletVehicle.class,
             ee.jakarta.tck.persistence.core.EntityGraph.Client.class,
+            ee.jakarta.tck.persistence.core.EntityGraph.ClientPuservletTest.class,
             com.sun.ts.tests.common.vehicle.VehicleRunnable.class,
             com.sun.ts.tests.common.vehicle.ejb3share.UserTransactionWrapper.class,
             com.sun.ts.lib.harness.EETest.class,

```

Can tck-rewrite-ant add the `ee.jakarta.tck.persistence.core.EntityGraph.ClientPuservletTest.class` class?  Or do we need a change elsewhere?

**Comment by starksm64** [comment]
This is certainly related to #133 because the assumption to date is that the generated test class is just a trigger to invoke the test through the javatest/appclient protocols. These really only need the legacy base test class to work. If we have generated tests that have logic to execute, then we need to both use the generated test class as the server side target and include it in the deployment artifiact. The only question I have is how to detect the different requirement.

**Comment by scottmarlow** [comment]
Just to confirm that I am no longer using the hack of adding class `ee.jakarta.tck.persistence.core.EntityGraph.ClientPuservletTest.class` which I mentioned in the description.  I closed #133 and will close this issue as well as they are no longer a problem.
